misc/rxgfix3

```diff
@@ -89,19 +89,22 @@ FILE: foreach $name (@ARGV) {
             }
         } else {
             if (!defined($opt_d)) {
-                print "Backup file $bak already exists, which would block processing.\n";
+                print "Backup file $bak already exists, which blocks processing.\n";
             } else {
-                print "Existing back-up file $bak: would be deleted\n";
+                print "Delete existing back-up file $bak\n";
             }
         }
     }
 }
 if ($bad_n) {
-    print "  Couldn't find some specified file(s).\n";
+    print "  Couldn't find file(s) listed above.\n";
 }
 if ($bad_b) {
-    print "  Some back-up file(s) already exist, delete or rename them.\n";
-    print "  You can delete them as part of the processing by adding the -d option.\n";
+    print
+"  Some back-up file(s), see messages above, already exist or couldn't
+  be deleted. Unless there an error deleting them, you can delete them
```

Comment:
"unless there ?is?was? an error" perhaps

Reply:
:) "is" I guess. This is the most complete one-time fix-up utility we have ever had. Hopefully, it will be useful.